### DISCUSSION TITLE
add popper-class for tooltip in select component(为el-tooltip 添加 popper-class)

### DISCUSSION THREAD

**Discussion by EmilyLike** [element-plus/element-plus#11544] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `select`

## Description

I would like to add class into inner tooltip in el-select component. 

问题场景（question situation）：
在使用select 组件中的collapse-tags-tooltip属性时，发现展开的tag宽度无限溢出。 本意想通过 css 设置 max-width 限定其宽度。 在查看其tooltip样式后，发现其默认class与其他tooltip 一致。
When I use the attribute collapse-tags-tooltip of select component, I find the width of popper tags tooltip is too wide. I happily think I  can use max-width(css attribute) to change the tooltip width. But when I check the specific tooltip css name, unlucky, I found the css name is normal , called "el_popper is-light", once I set max-width for this css name, all the popover width will change.

So I read the select component code, then , I found the el-tooltip didn't set specific class name.

解决思路(How to solve it )：
Basically, I think the easy method is to add a popper-class attribute on the tooltip. The classname could named as 'select-tags-tooltip'. Comparing to directly set max-width style on tooltip, setting popper-class could provide a more flexible opinion。 
Because the user could set different style by this css name for tooltip in select.
按照我的解决思路，可以在源码中设置 el-tooltip 的地方添加popper-class。classname 可设为：select-tags-tooltip。相比于直接在源码中设置max-width，这样的好处是定制性和通用性都能满足。

代码如下(packages/components/select)：
```
                <el-tooltip
                  v-if="collapseTagsTooltip"
                  :disabled="dropMenuVisible"
                  :fallback-placements="['bottom', 'top', 'right', 'left']"
                  :effect="effect"
                  placement="bottom"
                  popper-class="select-tags-tooltip"
                  :teleported="teleported"
                >
                  <template #default>
                    <span :class="nsSelect.e('tags-text')"
                      >+ {{ selected.length - 1 }}</span
                    >

```

### Additional comments

_(empty)_

**Comment by vampirefan** [comment]
该功能还没加上吗，我也遇到同样的问题，特别是选择项很多的时候，会自动撑满全宽。另一个极端是cascade组件，会自动撑满全高。
